crates/goose-cli/src/session/mod.rs

```diff
@@ -1079,7 +1078,7 @@ impl Session {
                                                         msg.to_string()
                                                     }
                                                 };
-                                                (formatted, Some(subagent_id.to_string()), Some(notification_type.to_string()))
+                                                (formatted, subagent_id.map(|s|s.to_string()), Some(notification_type.to_string()))
```

Comment:
no changes to notificaation type?

Reply:
yeah looking below, that should probably also not be coerced to Some